test/Integration/IntegrationTestBase.cs

```diff
@@ -107,7 +107,7 @@ private void SetupDatabase()
 
                 // Either use integrated auth or SQL login depending if SA_PASSWORD is set
                 string userId = "SA";
-                string password = Environment.GetEnvironmentVariable("SA_PASSWORD");
+                string password = "Yukon900"; // Environment.GetEnvironmentVariable("SA_PASSWORD");
```

Comment:
hardcoded password, please revert this back.

Reply:
Yeesh, thanks. Fixed!